client/src/components/generic/Footer/Footer.story.tsx

```diff
@@ -0,0 +1,25 @@
+import type { Meta, StoryObj } from "@storybook/react"
+
+import { Footer } from "./Footer"
+import { MemoryRouter } from "react-router-dom"
+
+const meta: Meta<typeof Footer> = {
+  component: Footer
+}
+
+export default meta
+type Story = StoryObj<typeof meta>
+export const DefaultFooter: Story = {
+  tags: ["autodocs"],
+  args: {
```

Comment:
Can we remove this? This might confuse ppl

Reply:
done